src/Illuminate/Http/Concerns/InteractsWithInput.php

```diff
@@ -91,6 +91,19 @@ public function has($key)
         return true;
     }
 
+    /**
+     * Determine if the request contains a given input item key.
+     *
+     * @param  string|array  $key
+     * @return bool
+     */
+    public function missing($key)
+    {
+        $keys = is_array($key) ? $key : func_get_args();
+
+        return ! $this->has($keys);
```

Comment:
What about this? Just forward arguments to has method.

```php
return ! $this->has(...func_get_args());
```

Reply:
That won't work if the argument is an array.

Reply:
@driesvints I am sure that it works properly. You can try this link https://repl.it/@QuynhNguyen/VapidImmaterialAccess

```php
<?php

class Request
{
  public function missing($key)
  {
    return $this->has(...func_get_args());
  }

  public function has($key)
  {
    $keys = is_array($key) ? $key : func_get_args();

    var_dump($keys);
  }
}

$request = new Request;

$request->missing('first'); // $keys = ['first']

$request->missing('first', 'second', 'third'); // $keys = ['first', 'second', 'third']

$request->missing(['first', 'second', 'third']); // $keys = ['first', 'second', 'third']
```